content/docs/manual/access-control.md

```diff
@@ -0,0 +1,201 @@
+---
+title: "Access Control"
+weight : 3900
+menu:
+  docs:
+    parent: manual
+---
+
+*NOTE: This documentation covers the Zenoh 1.0 ACL config. For Zenoh 0.11 ACL config,
+please refer to the [Zenoh 0.11 Access Control Rules RFC](https://github.com/eclipse-zenoh/roadmap/blob/ca841fe219890bf73289089b520271d70ded89b6/rfcs/ALL/Access%20Control%20Rules.md)*
+
+*Access control* enables Zenoh instances to filter (allow or deny) messages,
+depending on certain characteristics of individual messages and their respective source or destination.
+*Authentication* on the other hand allows Zenoh instances to identify certain characteristics in other instances they connect to,
+which are used to match the instances with configured subjects in the ACL policies and apply the rules accordingly on the exhanged messages.
+
+The configuration of access control policies is done via a [configuration file](../configuration).
+
+---------
+## ACL configuration
+
+ACL configuration mainly consists of three components: `rules`, `subjects` and `policies`, to which is added the `default_permission` (`allow` or `deny`) to be applied on messages that do not match the configured policies.
+
+The `enabled` boolean field allows to enable and disable ACL. Note that ACL config cannot be updated at runtime, and requires a restart of the instance to reflect the changes.
+
+Below is the example config we will be analyzing in this documentation.
+
+```json
+{
+  access_control: {
```

Comment:
access_control will need "" if this is json.

Reply:
All json in this page is json5. I'll update the format.